### ISSUE TITLE
FR: Add padding to the text box

### ISSUE THREAD

**Issue by hiforrest** [zsviczian/obsidian-excalidraw-plugin#1320]
I have been using excalidraw for a long time, and the most commonly used tool for me is the text box, which is absolutely essential. I would greatly appreciate it if you could adjust the paddings.

![Snipaste_2023-08-23_14-53-23](https://github.com/RainCat1998/obsidian-custom-attachment-location/assets/2953848/15b265e4-c800-4e87-942a-624429452102)

![Snipaste_2023-08-23_15-00-37](https://github.com/RainCat1998/obsidian-custom-attachment-location/assets/2953848/4b3ba3b0-af9e-4791-ad9c-e1af1b678cd3)

**Describe the solution you'd like**

![Snipaste_2023-08-23_15-10-18](https://github.com/RainCat1998/obsidian-custom-attachment-location/assets/2953848/4a9504b3-791b-4e25-a4cb-5ad4249fc598)

**Comment by zsviczian** [comment]
Yes, I find this problematic as well and it is also a relatively easy fix. The reason I am holding back on this is because this would be yet another feature that breaks compatibility with Excalidraw.com.

**Comment by hiforrest** [comment]
Could you possibly prioritize addressing this issue? After all, humans are visual creatures who appreciate aesthetics and order. I believe that resolving this small problem can greatly enhance the overall user experience and satisfaction towards Excalidraw. Your prompt attention to this request would be greatly appreciated.

**Comment by malteprinzler** [comment]
I would also really like this feature. Since I personally don't care about compatibility with excalidraw, I would like to implement the feature myself in a fork. @zsiviczian would you mind pointing me to the relevant piece of code so I can implement the change myself?
